yoyodyne/models/lstm.py

```diff
@@ -298,26 +296,39 @@ def forward(
             batch (data.PaddedBatch).
 
         Returns:
-            torch.Tensor: tensor of predictions of shape seq_len x
-                batch_size x target_vocab_size.
+            Union[Tuple[torch.Tensor, torch.Tensor], torch.Tensor]: beam
+                search return a tuple with a tensor of predictions of shape 
+                n x seq_len and tensor with the score for each prediction.
```

Comment:
Maybe replace "score" here with something more precise. Since it is common to normalize these scores by sequence length, we could say: "the unnormalized sum of symbol log-probabilities" or something like that?